roles/postgresql/defaults/main.yml

```diff
@@ -90,6 +90,10 @@ postgresql_hue_user: hue
 postgresql_hue_password: hue
 postgresql_hue_database: hue
 
+postgresql_exporter_user: postgres_exporter
```

Comment:
I think the user can be `exporter` without `postgres_` prefix. This is a PostgreSQL user so no need to add a prefix for me.